overall-equipment-effectiveness/workflows/Paper_Machine_Data_Simulator.yaml

```diff
@@ -0,0 +1,1969 @@
+resourceType: Flow
```

Comment:
This Device: Update Node appears to be updating a a system device from a different application, which I sure hope would fail ...

(I tried on staging and it does)

<img width="1026" alt="Screen Shot 2020-09-25 at 6 06 08 PM" src="https://user-images.githubusercontent.com/881769/94319998-de999200-ff59-11ea-91f1-d03971715e76.png">

Reply:
Miss when cloning the workflow - updated.